packages/Button/src/Button.js

```diff
@@ -3,7 +3,7 @@ import PropTypes from "prop-types";
 import RawButton from "@paprika/raw-button";
 import RefreshIcon from "@paprika/icon/lib/Refresh";
 import DownIcon from "@paprika/icon/lib/CaretDown";
-import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import { ShirtSizes } from "../../helpers/src/customPropTypes";
```

Comment:
Imagine `Button.js` is installed at `projects/client/node_modules/@paprika/button/lib/Button.js`.
```js
path.join('projects/client/node_modules/@paprika/button/lib', '../../helpers/src/customPropTypes');
// => 'projects/client/node_modules/@paprika/helpers/src/customPropTypes'
```
Doesn't work unless you also publish and install `@paprika/helpers`.